nixos/modules/services/security/tor.nix

```diff
@@ -715,7 +715,7 @@ in
               (submodule {
                 options = {
                   onion = mkOption {
-                    type = strMatching "[a-z2-7]{16}(\\.onion)?";
+                    type = strMatching "[a-z2-7]{16,56}(\\.onion)?";
```

Comment:
```suggestion
                    type = strMatching "([a-z2-7]{16}|[a-z2-7]{56})(\\.onion)?";
```

Reply:
Btw, do you know if `.onion` is actually optional? That's not mentioned on https://2019.www.torproject.org/docs/tor-manual.html.en, but that documentation is outdated anyway and doesn't even mention v3 addresses.